src/package-converter.coffee

```diff
@@ -162,7 +162,12 @@ class PackageConverter
         extension = path.extname(child)
         name = path.basename(child, extension)
 
-      selector = new ScopeSelector(scope).toCssSelector() if scope
+      try
+        selector = new ScopeSelector(scope).toCssSelector() if scope
+      catch e
+        e.fileName = path.join(sourceSnippets, child)
```

Comment:
The `fileName` property seems to be non-standard and only works in Firefox.  I'm not sure we need this considering the filename is printed anyway in the message.